<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEditSearch"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButtonSearch">
      <property name="text">
       <string>go</string>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QListWidget" name="listWidgetManga"/>
    </item>
    <item row="1" column="3">
     <widget class="QListWidget" name="listWidgetChapter"/>
    </item>
    <item row="1" column="4">
     <widget class="QListWidget" name="listWidgetBookmarks"/>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="labelCover">
      <property name="minimumSize">
       <size>
        <width>271</width>
        <height>181</height>
       </size>
      </property>
      <property name="text">
       <string>Cover Image</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="5">
     <widget class="QLabel" name="labelStatus">
      <property name="minimumSize">
       <size>
        <width>681</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButtonLastRead">
      <property name="text">
       <string>Last read</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4" colspan="2">
     <widget class="QPushButton" name="pushButtonDelete">
      <property name="text">
       <string>delete bookmark</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1082</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
